### ISSUE TITLE
3rd Party Install

### ISSUE THREAD

**Issue by keithmichelson** [angular/angular-cli#2063]
1. OS?  Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)
Mac OSX El Capitan

2. Versions. Please run `ng --version`. If there's nothing outputted, please run
in a Terminal: `node --version` and paste the result here:
angular-cli: 1.0.0-beta.11-webpack.8
node: 6.4.0
os: darwin x64

3. Repro steps. Was this an app that wasn't created using the CLI? What change did you
   do on your code? etc.
Brand new install running ng new with scss as default

4. The log given by the failure. Normally this include a stack trace and some
   more information.
jQuery required for bootstrap, not found

5. Mention any other details that might be useful.

**Comment by keithmichelson** [comment]
I figured this out, I guess
I'm using "bootstrap": "^4.0.0-alpha.4", but I guess it's not using jQuery 3 so I reverted it to "jquery": "^2.2.4", and it went away.

**Comment by deebloo** [comment]
you should be able to say ```import * as $ from jquery;```

**Comment by keithmichelson** [comment]
@deebloo Thanks for responding. I get a "String literal expected." error with that.

**Comment by deebloo** [comment]
"jquery"

**Comment by deebloo** [comment]
So. Import * as $ from "jquery";

**Comment by keithmichelson** [comment]
Oh haha duh. Thanks, but I still get a "Cannot find module 'jquery'." It's definitely in node_modules and the path is correct.

**Comment by deebloo** [comment]
Do you have the typings installed?

**Comment by keithmichelson** [comment]
Yeah, hopefully the correct one. I can see it in my typings folder. jquery.d.ts

**Comment by smrtsmrf** [comment]
I'm trying to get materialize-clockpicker in to my angular 2 app. I'm using the webpack 8 version of the cli. I'm trying to follow the steps on the angular cli github for installing 3rd party libraries. I'm uncertain on this line:

"Simply install your library via npm install lib-name --save and import it in your code"

What does it mean to import it into my code? What needs to go where? I've put the scripts in the angular-cli.json, but I get the error:

ERROR in multi scripts
Module not found: Error: Can't resolve '/Users/smrtsmrf/DevMtn/devmtn_scheduling/node_modules/materialize-clockpicker/dist/js/materialize-clockpicker.js' in '/Users/smrtsmrf/DevMtn/devmtn_scheduling/node_modules/angular-cli/addon/ng2/models'
 @ multi scripts

So I must be missing a step, but I don't know what I'm missing.

**Comment by keithmichelson** [comment]
Did you run this? npm i angular2-materialize --save
And then link to that location in the .json?

**Comment by smrtsmrf** [comment]
@keithmichelson What do you mean by "link to that location in the .json"? 
Also, the materialize-clockpicker doesn't say it needs angular2-materialize... ??
I tried adding the path to the js files ("../node_modules/materialize-clockpicker/dist/js/materialize.clockpicker.js") to the scripts property in angular-cli.json, but it threw error I posted above.

**Comment by keithmichelson** [comment]
Sorry, I wasn't sure what version or materialize you are using but you need to do an npm install of whatever one and make sure you have the correct path in angular-cli.json

npm i materialize-clockpicker --save

**Comment by filipesilva** [comment]
There are some instructions for using `bootstrap` in https://github.com/angular/angular-cli#global-library-installation.

**Comment by bunts94** [comment]
@smrtsmrf was your issue ever resolved because the same thing is happening to me?

**Comment by smrtsmrf** [comment]
@bunts94 Unfortunately I never did find a proper solution. What I did was install the package, then copy the needed js/css files from the node_modules folder and put them into my app explicitly. It works, but I feel it defeats the purpose.

**Comment by cb03037** [comment]
I am experiencing the same issue. Couldn't resolve jquery-mousewheel from script set in angular-cli.json. The malihu scrollbar was resoved but not the mousewheel. Mousewheel resolved finally after I closed down my command prompt and reserved the app .(close and reopen command prompt, run ng serve). I changed nothing else.
```
"scripts": [
        "../node_modules/malihu-custom-scrollbar-plugin/jquery.mousewheel.js",
        "../node_modules/malihu-custom-scrollbar-plugin/jquery.mCustomScrollbar.concat.min.js"
      ],
```

Seems like sometimes the compiler doesn't pickup new files/changes sometimes. 

`
angular-cli: 1.0.0-beta.16
node: 4.6.0
os: win32 x 64
`
